requirements.functest.txt

```diff
@@ -0,0 +1,23 @@
+#Install the agent
+-e .
+
+# OpenStack dependancies
+git+https://github.com/openstack/neutron@9.1.0
```

Comment:
Should be 'stable/liberty' instead of '9.1.0'.  Or is this the problem we've seen with an import failure on something like an invalid configuration option?